README.md

```diff
@@ -40,13 +40,12 @@ Also, you can of course not use some features outlined here. That's why the whol
 
 
 ## Feature list
-* easy to understand, easy to modify (main script with less than 120 LOC)
+* easy to understand, easy to modify
 * POSIX-compatible
-* logging-friendly
-* passphrase saved in external file, which you may protect in a better way
-* pruning included
-* privilege-separation (login scripts can have higher privilege than backup process)
```

Comment:
Leave it there, although the wording is obscure as users do likely not know what "login scripts" I mean. I get this and may change the wording, but for now leave it there…

If you want to change yourself. Login script is https://github.com/rugk/borg-cron-helper/blob/master/checklastbackup.sh